root/etc/e-smith/templates/etc/sudoers.d/30nethserver_cockpit_roles/10base

```diff
@@ -0,0 +1,99 @@
+{
+
+    use strict;
+    use warnings;
+    use JSON;
+    #use esmith::ConfigDB;
+
+
+    sub read_json
```

Comment:
remove read_json and use `require "/usr/libexec/nethserver/api/lib/auth_functions.pl";`